pkg/ccl/changefeedccl/changefeed_processors.go

```diff
@@ -1919,6 +1919,7 @@ func (cf *changeFrontier) manageProtectedTimestamps(
 		return false, nil
 	}
 
+	ptsStart := timeutil.Now()
```

Comment:
please make a new scoped timer in https://github.com/massimo-ua/cockroach/blob/e8834061faa086ca396c0081bb36165c0125dc8e/pkg/ccl/changefeedccl/timers/timers.go and use that instead